pkg/runner/lsds.go

```diff
@@ -0,0 +1,234 @@
+package runner
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"github.com/tass-io/scheduler/pkg/dto"
+	"github.com/tass-io/scheduler/pkg/span"
+	api "github.com/tass-io/tass-operator/api/v1alpha1"
+	"go.uber.org/zap"
+	"io/ioutil"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
I prefer a unified short name that I have seen in other projects, like

```go
metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
corev1 "k8s.io/api/core/v1"
```

For our tass project, I also use a unified name that simulates the above:

```go
serverlessv1alpha1 "github.com/tass-io/tass-operator/api/v1alpha1"
```